BookingsApi/BookingsApi/Controllers/HearingsController.cs

```diff
@@ -115,6 +115,36 @@ public async Task<IActionResult> GetHearingsByUsername([FromQuery]string usernam
             return Ok(response);
         }
 
+        /// <summary>
+        /// Get list of all hearings in a group
+        /// </summary>
+        /// <param name="groupId">the group id of the single day or multi day hearing</param>
+        /// <returns>Hearing details</returns>
+        [HttpGet(Name = "GetHearingsByGroupId")]
+        [OpenApiOperation("GetHearingsByGroupId")]
+        [ProducesResponseType(typeof(List<HearingDetailsResponse>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
```

Comment:
Your code doesn't return a 404, the query always returns an empty list. You should look to the same here IMO

Reply:
Controller code updated to return 404 when no hearing is found - this would make sense?